wallet/ios/Podfile

```diff
@@ -1,3 +1,6 @@
+#Override Firebase SDK Version
+$FirebaseSDKVersion = '9.2.0'
```

Comment:
Why?

Reply:
In iOS, pod files are not installing because of incompatible version of "FirebaseAppCheck" that is why I am overriding "FirebaseSDKVersion".